com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineSerializedCamera.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace UnityEditor.Rendering.Universal
 {
-    internal class UniversalRenderPipelineSerializedCamera : ISerializedCamera
+    class UniversalRenderPipelineSerializedCamera : ISerializedCamera
```

Comment:
why did we remove the internal keyword?

Reply:
`internal` is the default if no access modifier is specified, so I thought it was redundant.

Reply:
ok! thats fine then :)